app/models/ab_test_session.rb

```diff
@@ -0,0 +1,37 @@
+# reference document to map a user/metrics_uuid to a feature_flag A/B test group
+class AbTestSession
```

Comment:
I might push back here... `AbTest` sounds like the top-level event here, as in the feature being compared against the control.  `AbTestSession` to me implies "an instance of the A/B test" - specifically, it's an end user looking at something through a client that has been assigned to a group.  And while it may not have an explicit duration, the session as long as `enable_ab_test` is `true` for that feature flag.  I could easily see adding convenience methods on the session object like `enabled?` that would check if the parent flag is still in A/B test mode, or adding those timestamps to the model itself to know how long that session was live.  It didn't seem pertinent to do so, but it would be very easy to add.

You're right in that it doesn't apply to the convention, but just calling it `AbTest` lacks specificity in what the thing actually is in my mind.  Also my brain just reads that as "ab test" like some kind of PT exercise.

Reply:
I think it's reasonable for opinions to differ, especially since we're new to this!  I defer to you on the naming here.  

I'll just need to keep in mind that an "A/B test session" is mapped to users, not sessions, and pay attention to when we're talking about these types of sessions versus the more common type of sessions.

Reply:
Fair - that's a good point re: users.  Maybe then `AbTestAssignment` or `AbTestGroup`?

Reply:
Ah, either would be better.  Maybe `AbTestAssignment`?  I think that somewhat gets to both our concern.  I'd be fine with `AbTestGroup` too.

Reply:
[c651406](https://github.com/broadinstitute/single_cell_portal_core/pull/1721/commits/c6514069e252901a60cc08fc65cf82f9e769b0c5)